client/components/tinymce/plugins/media/image-orientation-utils.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import loadImage from 'blueimp-load-image';
+import 'blueimp-canvas-to-blob';
+import assign from 'lodash/assign';
+
+/**
+ * @returns { int } Orientation of the image
+ *
+ * @param { File } file --  The `window.File` object representing the image
+ */
+export function getImageOrientation( file ) {
+  return new Promise ( resolve => {
+    loadImage.parseMetaData( file, data => {
+      resolve( data.exif ? data.exif.get( 'Orientation' ) : -1 );
```

Comment:
Is it not sufficient to return a fallback of the default orientation `1`, or do we need to specially account for cases where it couldn't be determined?